.github/workflows/pull_request.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         os: [ubuntu-latest]
         java-version: [8, 11]
         maven-version: ['3.8.1']
-      fail-fast: true
+      fail-fast: false
```

Comment:
so this is not for final change, right? only to speed up this PR?

Reply:
I propose it for real yes, like in this case the error was just on java8 and java11 was cancelled due to this flag. But it's true it will consume more quota and it's probably covering rear cases, wdyt?

Reply:
if it fails always or very often, it is ok, i guess